ZuluContent/Items/Containers/TrapableContainer.cs

```diff
@@ -126,21 +117,14 @@ public virtual bool ExecuteTrap(Mobile from)
                     }
                     case TrapType.DartTrap:
                     {
-                        SendMessageTo(from, 502999, 0x3B2); // You set off a trap!
-
                         if (from.InRange(loc, 3))
                         {
-                            int damage;
-
-                            if (m_TrapLevel > 0)
-                                damage = Utility.RandomMinMax(5, 15) * m_TrapLevel;
-                            else
-                                damage = m_TrapPower;
+                            int damage = Utility.RandomMinMax(0, 15) * TrapStrength * 2;
```

Comment:
Since all the damage calculations are the same should we just defined this variable this once?

Reply:
Yeah I'll consolidate them, probably need to circle back to traps in general for #78